desktop/window-handlers/external-links/index.js

```diff
@@ -9,7 +9,7 @@ const debug = require( 'debug' )( 'desktop:external-links' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
```

Comment:
there would definitely be less friction if we change `lib/config` in wp-calypso to be `/lib/create-config`.